docs/qcreset.rst

```diff
@@ -0,0 +1,218 @@
+The qcreset module
+=====================
+
+The ``qcreset`` module provides methods (functions) to reset the selected data
+before starting a forward modelling workflow.
+
+
+Design philosophy
+-----------------
+
+* The client (user) scripts shall be small and simple and easy to use also
+  for modellers with little Python experience.
+* Input will be a python dictionary
+* The ``qcreset`` methods are meant to run inside RMS at the begining of
+  a modelling process
+
+
+set_data_empty
+--------------
+
+
+This method set all data described as input as empty. The data are not deleted
+so that the geometry of the data (surfaces or 3D grid properties) should be
+preserved and no workflow component (job or other) should be reset.
+
+
+Signature
+~~~~~~~~~
+
+The input of this method is a Python dictionary with defined keys. The key
+"project" is required while "horizons", "zones" and "grid_models" are optional
+(at least one of them should be provided for the method to have any effect).
+
+project
+  The roxar magic keyword ``project`` refering to the current RMS project.
+
+horizons
+  A Python dictionary where each key corresponds to the name of the horizons
+  category where horizon data need to be made empty. The value associated to
+  this key should be a list of horizon names to modify. If an empty list is
+  assigned, all available horizon names for this category will be used.
+
+zones
+  A Python dictionary where each key corresponds to the name of the zones
+  category where zone data need to be made empty. The value associated to
+  this key should be a list of zone names to modify. If an empty list is
+  assigned, all available zone names for this category will be used.
+
+grid_models
+  A Python dictionary where each key corresponds to the name of the grid models
+  where properties need to be made empty. The value associated to this key
+  should be a list of property names to modify. If an empty list is assigned,
+  all available zone names for this category will be used.
+
+
+
+Known issues
+~~~~~~~~~~~~
+
+* None for now
+
+
+Examples to run from RMS
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Example 1
+^^^^^^^^^
+
+.. code-block:: python
+
+    from fmu.tools import qcreset
+    import roxar
+
+    SETUP = {
+        "project": project,
+        "horizons": {
```

Comment:
I guess the average user will mostly empty an entire horizon folder or an entire grid, so maybe we should support giving in a list as well for usability? e.g "horizons": ["horizon_folder1", "horizon_folder2"]